tests/store_helpers.py

```diff
@@ -199,7 +199,7 @@ def add_application(
     @staticmethod
     def ensure_bed_version(store: Store, bed_name: str = "dummy_bed") -> BedVersion:
         """Return existing or create and return bed version for tests."""
-        bed: Optional[Bed] = store._get_query(table=Bed).filter(Bed.name == bed_name)
+        bed: Optional[Bed] = store._get_query(table=Bed).filter(Bed.name == bed_name).first()

```

Comment:
```suggestion
        bed: Optional[Bed] = store._get_query(table=Bed).filter(Bed.name == bed_name).first()

```

Use the filter function you added here, we do not want to run queries like this outside of the defined filters.

Reply:
I've created a `get_bed_by_name` method 👍